README.md

```diff
@@ -1,2 +1,55 @@
-# kex-push
-Dokumentation zum Push-Mechanismus für Vorgangsänderungen in KreditSmart
+# Push-Mechanismus für Vorgangsänderungen in KreditSmart
+
+Ein externer Client (public subscriber) registriert sich bei EUROPACE und erhält im Gegenzug Zertifikate, um damit eine Verbindung zu AWS aufnehmen zu können. Für Echtgeschäft und Testumgebung werden getrennte Zertifikate ausgestellt.
+
+Die Zertifikate erhalten Sie von Ihrem Ansprechpartner im **Kredit**Smart-Team. 
```

Comment:
entspricht das Fette Kredit noch der Norm? Bin mir nicht sicher ob das inzwischen nicht gekippt wurde.

Reply:
Laut [aktueller Auffassung](https://europace.slack.com/archives/C4K5LNPH9/p1523282953000167?thread_ts=1519650209.000128&cid=C4K5LNPH9) in normaler Schreibweise - hab's angepasst 👍